multimedia/welle.io/Portfile

```diff
@@ -5,8 +5,8 @@ PortGroup               qt5 1.0
 PortGroup               github 1.0
 PortGroup               cmake 1.1
 
-github.setup            AlbrechtL welle.io 801e483ce5340cb816cff63743706c245d8c5fe6
-version                 2.0-beta3+git801e483
+github.setup            AlbrechtL welle.io 2.0 v
+github.tarball_from     archive
```

Comment:
I believe that you need to add the epoch line so that people will see this update. Can you check on your end?

Reply:
Nice catch, epoch added, should be fine now and welle.io is now listed in `port outdated`